requirements.txt

```diff
@@ -11,7 +11,7 @@ PyQt5 == 5.12.2
 pyserial ~= 3.4
 
 # https://packages.ubuntu.com/bionic/python3-numpy
-numpy ~= 1.13
+numpy == 1.16.2
```

Comment:
Please specify explicitly, in a comment nearby, why this version is needed, and provide links to appropriate resources.